src/app/main/employees-sheet/sheets-list/sheets-list.component.ts

```diff
@@ -8,12 +11,35 @@ import { SheetFormComponent } from '../sheet-form/sheet-form.component';
   styleUrls: ['./sheets-list.component.scss']
 })
 export class SheetsListComponent implements OnInit {
-
+  paginationPerPage: number = 10
+  params: any = {
+    page: 1,
+    per_page: this.paginationPerPage,
+  };
+  paginationData: PaginationData;
```

Comment:
`paginationData: PaginationData;`
what does this means?

Reply:
it's an interface to meta data 

export interface PaginationData {
  current_page: number;
  next_page: number;
  prev_page: number;
  total_count: number;
  total_pages: number
}